README.md

```diff
@@ -26,7 +26,7 @@ Follow the [guide](https://docs.vmware.com/en/VMware-RabbitMQ-for-Kubernetes/1/r
 * Create a [GemFire cluster](https://docs.vmware.com/en/VMware-GemFire-for-Kubernetes/2.3/gf-k8s/create-and-delete.html)
```

Comment:
**[OVERALL-README-COMMENTS]**

**ONE:** Is Docker the only way to run locally?

**TWO:** Is there a way to stub out the required services when running in Docker/locally? This could be helpful for local manual testing as well as integration testing.

Reply:
`Kubernetes` is the another (and preferred) way to run the microservices locally.

Reply:
> Kubernetes is the another (and preferred) way to run the microservices locally.
 
Sure. However, it is nice to not to have to run k8s to manually test (a portion of the functionality) by using local services/stubs, etc.. 

As it is currently, if I were to stumble upon this repo I would try to **not** work on it because of the initial setup required w/ Gemfire and k8s etc.. 

Spring Cloud Dataflow is very similar to this scenario. However, for most bugs I can simply run `java -jar ...` and get a local app running and fix my bug etc.. Something to think about later for sure.